dev-util/clippy/clippy-9.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="frr-${PV}"
+PYTHON_COMPAT=( python3_{9..12} )
```

Comment:
Remove 3.9 while at it. This will trigger a `pkgcheck` warning, since it is no longer supported.

Reply:
Done